core/server/proxy/src/main/java/alluxio/proxy/PathsRestServiceHandler.java

```diff
@@ -110,7 +111,7 @@ public Response createDirectory(@PathParam("path") final String path,
     return RestUtils.call((RestUtils.RestCallable<Void>) () -> {
       if (options == null) {
         mFileSystem.createDirectory(new AlluxioURI(path),
-                CreateDirectoryPOptions.getDefaultInstance());
+                FileSystemOptions.createDirectoryDefaults(ServerConfiguration.global()));
```

Comment:
This goes to FileSystem. Why do you need to supply the defaults here?

Reply:
Same for all.

Reply:
Good catch. I changed this before we did the merging. Fixed